src/array.test.js

```diff
@@ -72,5 +72,95 @@ describe("Test includes() function", () => {
     expect(testArr.myIncludes(3, -1)).toEqual(true)
     expect(testArr.myIncludes(3, -100)).toEqual(true)
     expect(testArr.myIncludes(1, -2)).toEqual(false)
+  }) 
+})
+
+
+describe("Test some() function", () => {
+  it("Able to pass test using given function", () => {
+    const testArr = [1, 2, 3, 5, NaN]
+    const isEven = x => x % 2 === 0
+    const expectedRes = testArr.some(isEven)
+    expect(testArr.mySome(isEven)).toEqual(expectedRes) // True
+  })
+
+  it("Able to ignore undefined", () => {
+    const testArr = [,,,,,]
+    const isNaN = x => x === NaN
```

Comment:
NaN isn't the same as undefined. You would need separate test when testing for NaN